packages/storage/src/implementation/backoff.ts

```diff
@@ -49,10 +49,13 @@ export function start(
   }
   let triggeredCallback = false;
 
-  function triggerCallback(): void {
+  // TODO: This disable can be removed and the 'ignoreRestArgs' option added to
+  // the no-explicit-any rule when ESlint releases it.
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  function triggerCallback(...args: any[]): void {
```

Comment:
Should we follow the same pattern in handler() and change the signature as well? Something like:

```
function handler(success:boolean, ...args:any[])
```

Reply:
Done.